serveradmin/servershell/static/js/servershell/autocomplete/search.js

```diff
@@ -16,8 +16,7 @@ $(document).ready(function () {
 
     let term_input = $('#term');
     term_input.autocomplete({
-        delay: 50, // Wait n ms before starting to auto complete to avoid needles requests to backend
-        minLength: 0,
```

Comment:
what's the default minLength? Good that it's not `minLength: 0` anymore. ^^

Reply:
The default value is 1. I think 0 had a purpose for users knowing there is no auto completion. They just enter the field and immediately got some help but this is probably better.